### ISSUE TITLE
About adding localization translation support for `flet.dev` website

### ISSUE THREAD

**Issue by LaoshuBaby** [flet-dev/website#40]
Hello every developer in flet-dev, here is a proposal about allow translation for flet's homepage.

Today's web may meet with localization problem in many situation, if their user from all over the world. Some project mainly maintained by some company will use English only, such as google's [go.dev](https://go.dev) or [dart.dev](https://dart.dev) , while another popular programming language like rust, ruby, nodejs and even vue.js, all of them are becoming more friendly to user from different countries or region. Here I'll use S-Chinese as example:

* https://github.com/ruby/www.ruby-lang.org/tree/master/zh_cn
<img alt="ruby" src="https://user-images.githubusercontent.com/42690037/192034884-46557a04-6bbe-4c84-9531-731ab997f92b.png" width=350px/>

* https://github.com/rust-lang/www.rust-lang.org/tree/master/locales/zh-CN
<img alt="rust" src="https://user-images.githubusercontent.com/42690037/192035012-e138262b-0d66-43a1-9904-995b7bd773d9.png" width=350px/>

* https://github.com/nodejs/nodejs.org/tree/main/locale/zh-cn
<img alt="nodejs" src="https://user-images.githubusercontent.com/42690037/192035116-9f27c237-a01d-4bd2-8893-5583e874ce19.png" width=350px/>

* https://github.com/vuejs-translations/docs-zh-cn
<img alt="vue.js" src="https://user-images.githubusercontent.com/42690037/192035257-febe43e5-feda-4f12-a7ac-6c9646d5fecb.png" width=350px/>

Maybe you'll feel strange when just see those square character. For me, my language is S-Chinese, I can read English and write a piece of text, but if I can explore flet or read the docs in my native language, it will be happy and more relaxable, more willing to learn this framework, and let me know that at least one developer who using my native language has been active on this open source project.

I looks for flet-dev organization's current 6 repository and haven't found where to raise a proposal, so I open this issue, and want to listen to flet's maintainer's advice. Thank you for reading this.

**Comment by FeodorFitsner** [comment]
Thanks for reaching out! Do you want to help with translating Flet website to Chinese?

**Comment by LaoshuBaby** [comment]
> Do you want to help with translating Flet website to Chinese?

Of course, since I will open this issue, it obviously means that I am willing to be the translator of the website UI and docs. What I care about is when can I start and where I commit to.

**Comment by FeodorFitsner** [comment]
Sounds great!

So, Flet website is built with Docusaurus.
There is a guide for adding localizations in Docusaurus: https://docusaurus.io/docs/next/i18n/introduction
Also, there was an attempt to make Thai translation - you can use it as a template/sample: https://github.com/flet-dev/website/pull/9/files

Let me know if you have any questions!
